website/client/components/groups/chat.vue

```diff
@@ -233,6 +284,84 @@
       display: inline-block;
       width: 100px;
     }
+
+    .chat-count {
+      margin-top: 8px;
+      display: block;
+    }
+  }
+
+  .v-tribute {
+    width: 100%;
+  }
+</style>
+
+<style lang="scss">
+  @import '~client/assets/scss/tiers.scss';
+  @import '~client/assets/scss/colors.scss';
+
+  .tribute-container {
+    position: absolute;
+    top: 0;
+    left: 0;
+    height: auto;
+    max-height: 400px;
+    max-width: 600px;
+    overflow: auto;
+    display: block;
+    z-index: 999999;
+    border-radius: 4px;
+    box-shadow: 0 1px 4px rgba(#000, 0.13);
+  }
+  .tribute-container ul {
+    margin: 0;
+    margin-top: 2px;
+    padding: 0;
+    list-style: none;
+    background: #fff;
+    border-radius: 4px;
+    border: 1px solid rgba(#000, 0.13);
+    background-clip: padding-box;
+    overflow: hidden;
+    -moz-transition: none;
```

Comment:
it's not necessary to add browser-specific rules, we have a a plugin that automatically adds them when needed, for example in this case `transition: none;` alone will do the trick